test/unit/object/test_events.py

```diff
@@ -253,7 +273,7 @@ def test_generate_events_with_long_polling(
         empty_events_response,
     ]
     events = test_events.generate_events_with_long_polling(**stream_type_kwargs)
-    assert next(events) == mock_event
+    assert next(events) == mock_event_object
```

Comment:
instead of having the mock_event_object fixture you could construct the mock Event here based on the mock_event json.  Then do the assert check for equality.  Then I think you can remove the mock_event fixture.